ms-common-auth/src/main/java/net/trajano/ms/authz/Authorization.java

```diff
@@ -0,0 +1,28 @@
+package net.trajano.ms.authz;
+
+import net.trajano.ms.Microservice;
+import net.trajano.ms.authz.internal.CacheNames;
+import org.springframework.cache.annotation.EnableCaching;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+import javax.ws.rs.core.Application;
+
+/**
+ * Authorization token endpoint.
+ */
+@Configuration
+@EnableCaching
+@EnableScheduling
+@ComponentScan
+public class Authorization {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)